ncm-grub/src/main/pan/components/grub/schema.pan

```diff
@@ -11,6 +11,14 @@ include 'pan/types';
  line, where the first field matches the file_user parameter.
 }
 
+type grub_argument = {
+    @{a value for the argument (creates key=value when present; only key is generated when missing
+      (a list is converted in comma-separated list of strings).}
+    "value" ? element with is_string(SELF) || is_long(SELF) || is_list(SELF)
+    @{if false, do not add/update this argument and remove this option from the current list}
```

Comment:
Does this mean, if the argument already exists (i.e. the OS provides it by default) it will removed?

Reply:
not sure what you exactly ask, but it's the same behaviour as the preceding with a `-` in the args case